_shared/project/bin/make_devdata

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+import subprocess
+import tempfile
+from pathlib import Path
+
+if __name__ == "__main__":
+    with tempfile.TemporaryDirectory() as tmpdirname:
+        git_dir = Path(tmpdirname) / "devdata"
+        subprocess.check_call(
+            [
+                "git",
+                "clone",
+                "--depth",
+                "1",
+                "git@github.com:hypothesis/devdata.git",
+                git_dir,
+            ]
+        )
+        devdata_file = git_dir / "{{ cookiecutter.slug }}" / "devdata.env"
+        devdata_file.replace(Path(__file__).parent.parent / ".devdata.env")
```

Comment:
I tidied up this script slightly from the one in `pyramid-googleauth`: used `pathlib` instead of `os` and `shutil` and also added `--depth 1` to the `git clone` to speed it up (since we don't need the history)